src/__tests__/links.integration.test.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable complexity */
+const { extractLinksRecursively } = require('../utils/extractLinksRecursively');
+
+test('the build should not break any links', () => {
+  const uniqueLinks = extractLinksRecursively('_site');
```

Comment:
@AndyClifford does this contain all the astro and jekyll links?

Reply:
Yep all of the unique links and anchor tags on each page